### PR TITLE
zh-CN: android.md

### PR DESCRIPTION
**Pull request by Bryan-Kan**
Translate android.md

### CODE REVIEW

File: site/zh-cn/lite/guide/android.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+有关源代码的说明，您也应该阅读
+[TensorFlow Lite Android 图像分类](https://www.tensorflow.org/lite/models/image_classification/android).
+
+这个例子使用了
+[图像分类](https://www.tensorflow.org/lite/models/image_classification/overview)
+来将设备的后置摄像头拍到的东西进行持续分类。
+该应用程序可以在设备或者模拟器上都可以运行。
+
+推断(Inference)通过TensorFlow Lite的Java API来实现。演示应用程序可以对实时帧进行分类，并显示最有可能的类别。应用程序可以允许用户选择浮点或
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]: *Inference* 一般翻译为：推理

File: site/zh-cn/lite/guide/android.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+作为起始点。
+
+下面的部分包含了一些有关如何在Android上使用TensorFlow Lite的有用的信息。
+
+### 使用JCenter中的TensorFlow Lite AAR
+
+如果您要在您的Android应用程序中使用TensorFlow Lite，我们推荐使用
+[在JCenter中托管TensorFlow Lite AAR](https://bintray.com/google/tensorflow/tensorflow-lite)。
+
+您可以像下面这样写您的`build.gradle` 依赖:
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]: 原文 *You can specify this ...*，建议把***specify this***翻译出来：
```suggestion
您可以像下面这样在您的`build.gradle`依赖中指定它:
```

File: site/zh-cn/lite/guide/android.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+```build
+dependencies {
+    implementation 'org.tensorflow:tensorflow-lite:0.0.0-nightly'
+}
+```
+
+这个AAR包含了
+[Android ABIs](https://developer.android.com/ndk/guides/abis)中的所有的二进制文件。您可以通过只包含您需要的ABIs支持来减少您应用程序的二进制文件大小。
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]:
```suggestion
[Android ABIs](https://developer.android.com/ndk/guides/abis)中的所有的二进制文件。您可以通过只包含您需要支持的ABIs来减少您应用程序的二进制文件大小。
```

File: site/zh-cn/lite/guide/android.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+我们推荐大部分的开发者忽略 `x86`，`x86_64`，和`arm32` 的ABIs。您可以通过如下的Gradle配置实现，这个配置只包括了  `armeabi-v7a`和`arm64-v8a`，使用该配置会涵盖了大部分的主流Android设备。
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]: *modern* 翻译成 *现代* 比较好一点：
```suggestion
我们推荐大部分的开发者删简 `x86`，`x86_64`，和`arm32` 的ABIs。您可以通过如下的Gradle配置实现，这个配置只包括了 `armeabi-v7a`和`arm64-v8a`，该配置能涵盖住大部分的现代Android设备。
```

File: site/zh-cn/lite/guide/android.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+android {
+    defaultConfig {
+        ndk {
+            abiFilters 'armeabi-v7a', 'arm64-v8a'
+        }
+    }
+}
+```
+
+如果需要了解更多有关的 `abiFilters`的信息, 请查看Android Gradle文档中的
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]: 语句更简练一些：
```suggestion
想要了解更多有关 `abiFilters`的信息, 请查看Android Gradle文档中的
```

File: site/zh-cn/lite/guide/android.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+[`NdkOptions`](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.NdkOptions.html)。
+
+### 在本地构建TensorFlow Lite
+
+在某些情况下，您可能希望使用一个本地构建的TensorFlow Lite. 比如说，您可能正在构建一个自定义的二进制文件其中包含了
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]:
```suggestion
在某些情况下，您可能希望使用一个本地构建的TensorFlow Lite. 比如说，您可能正在构建一个自定义的包含了
```

File: site/zh-cn/lite/guide/android.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+[从TensorFlow中选择的操作](https://www.tensorflow.org/lite/guide/ops_select).
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]:
```suggestion
[从TensorFlow中选择的操作](https://www.tensorflow.org/lite/guide/ops_select)的二进制文件。
```

File: site/zh-cn/lite/guide/android.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+在这个例子中，请参照
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]: 这里的 *case* 指代的是 本地构建这种 *“情景、情况”* ，不是 *“例子”* ：
```suggestion
在这种情况下，请参照
```

File: site/zh-cn/lite/guide/android.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
+# Android 快速上手
+
+在您开始在Android上使用TensorFlow Lite之前，我们推荐您探索下面的例子。
+
+<a class="button button-primary" href="https://github.com/tensorflow/examples/tree/master/lite/examples/image_classification/android">Android
+图像分类例子</a>
+
+有关源代码的说明，您也应该阅读
```
Comment (yantaozhao) [inline site/zh-cn/lite/guide/android.md]: 建议把 *也* 改成 *还*：
```suggestion
有关源代码的说明，您还应该阅读
```

**Comment by lamberta** [comment]
@Bryan-Kan It looks like these commits are using a different email address than what your CLA is under. Can you re-sign the CLA so the bot doesn't get confused 🤖: https://cla.developers.google.com/
Thanks

**Comment by lamberta** [comment]
Confirmed CLA for user @Bryan-Kan 
CLA bot is unable to verify since CLA was signed with gmail address and commit was authored with outlook address. You might need to update your `.gitconfig` file.
Thank you!

**Pull request merged**